src/app/GitUI/CommandsDialogs/FormStash.cs

```diff
@@ -286,18 +286,17 @@ private void LoadGitItemStatuses(IReadOnlyList<GitItemStatus> gitItemStatuses)
             Loading.Visible = false;
             Loading.IsAnimating = false;
             Stashes.Enabled = true;
-            refreshToolStripButton.Enabled = gitStash == _currentWorkingDirStashItem;
         }
 
         private void ResizeStashesWidth()
         {
-            Stashes.Size = new Size(toolStrip1.Width - 15 - refreshToolStripButton.Width - showToolStripLabel.Width, Stashes.Size.Height);
+            Stashes.Width = toolStrip1.Width - 5 - showToolStripLabel.Width;
```

Comment:
It would great to document what "5" is.

Reply:
Adding a `const`. I had just removed the `refreshToolStripButton` and its spacing.
Please do not blame me for the layout of the rather old FormStash nor for limitations of widgets.
ToolStripItems do not have neither alignment "fill" nor public properties `Left` and `Right`.